policy/intrinsic_zernikes/comcam/intrinsic_zk__31_23.yaml

```diff
@@ -0,0 +1,194 @@
+--- 
+         
+ # TODO: This file needs to be updated once batoid allows for its generation
+ # for now this is just the same as comcam/intrinsic_zk_g_31_23.yaml
+ 
+ # Intrinsic Zk for the REF band 500 nm.         
+ # The first dimension is kmax = 31, corresponds to number of Zernike         
+ # polynomials used to measure wavefront variation accross the pupil.         
+ # The second dimension is jmax = 23, corresponds to number of Zernike         
```

Comment:
The second dimension is jmax = 23 --> The second dimension has the jmax = 23

This applies to all the `yaml` files.